src/mixins/filing-template-mixin.ts

```diff
@@ -514,7 +514,8 @@ export default class FilingTemplateMixin extends Vue {
     this.setNameRequest({
       legalType: businessSnapshot.businessInfo.legalType,
       legalName: businessSnapshot.businessInfo.legalName,
-      foundingDate: businessSnapshot.businessInfo.foundingDateTime
+      // *** TODO: Founding Date is not needed in Name Request object???
```

Comment:
Any idea who worked on this and can answer this question?

Reply:
Not sure about this

Reply:
I quickly scanned the code / getters and didn't see this property being used w.r.t. NRs.

It doesn't hurt functionality to leave this in, but it's a "decoy pattern" -- looks like it does something but doesn't -- which may confuse a developer (like me, again, in 2 weeks). 🦆